book/04-git-server/sections/git-daemon.asc

```diff
@@ -20,7 +20,43 @@ $ git daemon --reuseaddr --base-path=/srv/git/ /srv/git/
 If you're running a firewall, you'll also need to punch a hole in it at port 9418 on the box you're setting this up on.
 
 You can daemonize this process a number of ways, depending on the operating system you're running.
-On an Ubuntu machine, you can use an Upstart script.
+
+systemd being the most common init system among modern Linux distributions (Debian, Ubuntu, RHEL, CentOS…), you can use systemd for that purpose. Your systemd file should be places as follows:
```

Comment:
Let's make this a bit more prosey:

```
Since `systemd` is the most common init system among modern Linux distributions, you can use it for that purpose.
Simply place a file in `/etc/systemd/system/git-daemon.service` with these contents:
```